packaged/shared.sh

```diff
@@ -57,3 +57,19 @@ function get_lisk_custom_config() {
 	fi
 	echo "$LISK_CUSTOM_CONFIG"
 }
+
+# Default value of LISK_CUSTOM_CONFIG
+LISK_CUSTOM_CONFIG=$( get_lisk_custom_config "$PM2_CONFIG")
```

Comment:
Is `PM2_CONFIG` set here?

Reply:
Whoops, fixed

Reply:
Actually, need to double-check something....

Reply:
Nope, not quite ripe yet